.github/workflows/lint_and_unit_test.yml

```diff
@@ -17,7 +16,7 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v3
         with:
-          node-version: current
+          node-version-file: ".nvmrc"
```

Comment:
This works pretty nicely and is one less thing we'll need to update the next time we upgrade our Node version.

From the workflow logs:
```
Node version file is not JSON file
Resolved .nvmrc as lts/hydrogen
Attempt to resolve LTS alias from manifest...
Found in cache @ /opt/hostedtoolcache/node/18.16.0/x64
```